bifrost/app/blog/blogs/test-your-llm-prompts/src.mdx

```diff
@@ -0,0 +1,286 @@
+- How to properly test your prompts
+- How to evaluate prompts before and after deployment
+  - Method 1: Real user feedback
+  - Method 2: Human evaluators
+  - Method 3: LLM-as-a-judge
+- Why prompt evaluation is important
+- Examples
+
+Let's dive in!
+
+---
+
+# 5 best prompt experiment tools for building AI apps
+
+We hand-picked a few tools that support all of the following, along with comprehensive logging, dataset export capabilities and detailed performance tracking: **<span style={{color: '#0ea5e9'}}>Helicone, Langfuse, Arize AI, Braintrust and LangSmith.</span>** Here's a quick comparison:
+
+|                | **Data collection (Datasets)** | **Prompt Management** | **Playground** | **Evaluations** | **Experiments** | **Open-source** |
+| -------------- | ------------------------------ | --------------------- | -------------- | --------------- | --------------- | --------------- |
+| **Helicone**   | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ✔️              |
+| **Langfuse**   | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ✔️              |
+| **Arize AI**   | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ✔️              |
+| **Braintrust** | ✔️                             | ✔️                    | ✔️             | ✔️              | ✔️              | ❌              |
+| **LangSmith**  | ✔️                             | ✔️                    | ✔️             | ✔️              | ❌              | ❌              |
+
+## Components to improve LLM prompts
+
+### Data collection
+
+You need high-quality input variables to create high-quality prompts. A common practice is to use “**<span style={{color: '#0ea5e9'}}>golden datasets</span>**” - data that’s meticulously cleaned and labeled - to tune your prompt for problems that are easily reproducible.
+
+Complementing this approach, teams like QA Wolf also use **<span style={{color: '#0ea5e9'}}>randomly sampled production data</span>**, which represents actual user interactions and offers a more accurate representation of real-world use cases. QA Wolf uses Helicone as their observability tool to log every LLM request and export comprehensive datasets that can be used to further fine-tune their LLMs.
+
+### Prompt management
+
+Prompt registries are specialized tools to help engineers **<span style={{color: '#0ea5e9'}}>version, manage or improve prompts</span>**, and roll back to a previous version easily.
+
+### Prompt playgrounds
+
+Playgrounds are **<span style={{color: '#0ea5e9'}}>interactive environments</span>** that helps engineers iteratively test and refine their prompts. Notably, playgrounds allows you to view detailed prompt/response pairs, create and iterate on prompt templates, experiment with model configurations and test prompts in real-time.
```

Comment:
Playgrounds are **<span style={{color: '#0ea5e9'}}>interactive environments</span>** that helps engineers iteratively test and refine their prompts. Notably, playgrounds enables rapid prompt iteration, testing against real-world data, and fine-tuning of model parameters and metadata configurations like top_k.

Something like this is preferred